code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -795,6 +795,7 @@
 						entry.limit--
 					uses -= entry.cost
 			update_static_data(wizard) //update statics!
+			return
```

Comment:
this should return `TRUE` as it needs to update, no? Unless i'm missing something

Reply:
yep, looks like the tgui is old and never returned true, probably autoupdates too